1672910445_switcher_card.up.sql

```diff
@@ -0,0 +1,6 @@
+CREATE TABLE switcher_cards (
+  id TEXT PRIMARY KEY ON CONFLICT REPLACE,
+  type INT NOT NULL DEFAULT 0,
+  clock INT NOT NULL,
+  extra TEXT DEFAULT ""
```

Comment:
In the case of community cards, the id will correspond to the community id even when any channel is opened (only one card per community). So we need an extra field to store channel-id to open the corresponding channel on card press.
And similarly, for wallets, with only one card & one id. And `extra` field will be used to store which screen in that account.

Reply:
Maybe extra field is not self explanatory, open for suggestions

Reply:
Maybe `metadata` could be a better name? It's just as generic, but at least to me it implies that its *text* could be a serialized data structure.

Reply:
We are only going to store the id of a screen which card going to open in the extra field. So to avoid confusion and keeping fields flat I am thinking of renaming `id` to `cardId` and `extra` to `screenId`. wdyt?

Reply:
I just commented on renaming to `metadata` because I got the impression we don't know what we're going to store in the future, but I prefer your suggestion a lot more :)

Reply:
thank you